src/main/asciidoc/batch/file-ingest-sftp/local.adoc

```diff
@@ -0,0 +1,209 @@
+[[sftp-file-ingest-local]]
+==== Using the Local Server
+
+===== Additional Prerequisites
+
+* A running local Data Flow Server
+include::{docs_dir}/local-server.adoc[]
+
+* Running instance of link:http://kafka.apache.org/downloads.html[Kafka]
+* Either a remote or local host accepting SFTP connections.
+
+* A database tool such as link:https://dbeaver.jkiss.org/download/[DBeaver] to inspect the database contents
+
+NOTE: To simplify the dependencies and configuration in this example, we will use our local machine acting as an SFTP server.
+
+===== Building and Running the Demo
+
+. Build the demo JAR
++
+From the root of this project:
++
+```
+$ cd batch/file-ingest
+$ mvn clean package
+```
+NOTE: For convenience, you can skip this step.
+The jar is published to the  https://repo.spring.io/libs-snapshot-local/io/spring/cloud/dataflow/ingest/ingest/1.0.0.BUILD-SNAPSHOT/[Spring Maven repository]
++
+
+. Create the data directories
++
+Now we create a remote directory on the SFTP server and a local directory where the batch job expects to find files.
++
+NOTE: If you are using a remote SFTP server, create the remote directory on the SFTP server.
++
+```
+$ mkdir -p /tmp/remote-files /tmp/local-files
+```
++
+
+. Register the `sftp-dataflow` source and the `task-launcher-dataflow` sink
++
+With our Spring Cloud Data Flow server running, we register the `sftp-dataflow` source and `task-launcher-dataflow` sink.
+The `sftp-dataflow` source application will do the work of polling the remote directory for new files and downloading them to the local directory.
+As each file is received, it emits a message for the `task-launcher-dataflow` sink to launch the task for that file.
++
+In the Spring Cloud Data Flow shell:
++
+[source,console,options=nowrap]
+----
+dataflow:>app register --name sftp --type source --uri maven://org.springframework.cloud.stream.app:sftp-dataflow-source-kafka:2.0.3.BUILD-SNAPSHOT
+Successfully registered application 'source:sftp'
+dataflow:>app register --name task-launcher --type sink --uri maven://org.springframework.cloud.stream.app:task-launcher-dataflow-sink-kafka:1.0.0.BUILD-SNAPSHOT
+Successfully registered application 'sink:task-launcher'
+----
++
+. Register and create the file ingest task. If using the published jar set `--uri maven://io.spring.cloud.dataflow.ingest:ingest:1.0.0.BUILD-SNAPSHOT`:
+[source,console,options=nowrap]
+dataflow:>app register --name fileIngest --type task --uri file:///path/to/target/ingest-X.X.X.jar
+Successfully registered application 'task:fileIngest'
+dataflow:>task create fileIngestTask --definition fileIngest
+Created new task 'fileIngestTask'
++
+. Create and deploy the stream
++
+Now lets create and deploy the stream which will start polling the SFTP server and when new files arrive launch the batch job.
```

Comment:
> Now lets create and deploy the stream which will start polling the SFTP server and when new files arrive launch the batch job.

Now let's create and deploy the stream which when deployed, it will start polling the SFTP server, and when new files arrive, it will launch the batch job.